envision/server.py

```diff
@@ -421,6 +422,7 @@ class ModelFileHandler(FileHandler):
     def initialize(self):
         # We store the resource filenames as values in `path_map` and route them
         # through `importlib.resources` for resolution.
+        # pytype: disable=wrong-arg-types
```

Comment:
This should be possible to fix by changing the signature of the base to the following: `FileHandler.initialize(path_map: Dict[str,Union[str, Path]]`.